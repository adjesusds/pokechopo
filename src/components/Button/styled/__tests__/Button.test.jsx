// External Packages
import React from 'react'
import { create } from 'react-test-renderer'

// Internal Packages
import StyledButton from '../Button'

describe('<StyledButton />', () => {
  it('should style the component on valid props', () => {
    let styledComponent

    styledComponent = create(<StyledButton />)

    expect(styledComponent).toMatchSnapshot()

    styledComponent = create(<StyledButton disabled />)

    expect(styledComponent).toMatchSnapshot()
  })
})
