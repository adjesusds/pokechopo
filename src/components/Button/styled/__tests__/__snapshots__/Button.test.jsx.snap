// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StyledButton /> should style the component on valid props 1`] = `
.c0 {
  background-color: #000000;
  border: 2px solid #ffffff;
  border-radius: 3px;
  color: #ffffff;
  cursor: pointer;
  padding: 1rem;
  -webkit-transition: all 200ms ease-in;
  transition: all 200ms ease-in;
  -webkit-transition-property: border-color,color;
  transition-property: border-color,color;
}

.c0:focus {
  outline: none;
}

<button
  className="c0"
/>
`;

exports[`<StyledButton /> should style the component on valid props 2`] = `
.c0 {
  background-color: #000000;
  border: 2px solid #ffffff;
  border-radius: 3px;
  color: #ffffff;
  cursor: pointer;
  padding: 1rem;
  -webkit-transition: all 200ms ease-in;
  transition: all 200ms ease-in;
  -webkit-transition-property: border-color,color;
  transition-property: border-color,color;
  border-color: #9e9e9e;
  color: #9e9e9e;
  cursor: wait;
}

.c0:focus {
  outline: none;
}

<button
  className="c0"
  disabled={true}
/>
`;
